hrjobcontract/js/test/services/contract-leave_test.js

```diff
@@ -29,9 +28,19 @@ define([
     });
 
     describe('getOne()', function () {
+      var expected;
```

Comment:
The name of this variable does not seem to be self-explanatory. What does it store?